lib/Doctrine/ORM/Query/ResultSetMapping.php

```diff
@@ -428,7 +430,7 @@ public function isScalarResult($columnName)
      *
      * @param string $alias
      *
-     * @return string
+     * @return class-string
```

Comment:
This is not understood by IDEs is it?

Reply:
It's now an alias of string in PHPStorm. Not sure in VS Code